### PR TITLE
Suggest commands if incorrect entered

### PR DESCRIPTION
**Pull request by piyush-garg**
# Changes

This will add the feature of suggestion commands available
if wrong command provided.

Current output is like
`tkn pr descri`
`Error: unknown command "describ" for "tkn pipelinerun"`

After this patch
`tkn pr descri`
`Error: Use one of available subcommands: describe, list, logs`

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/cli/blob/master/CONTRIBUTING.md)
for more details._

# Release Notes

```
release-note
```

### CODE REVIEW

**Comment by sthaha** [comment]
@piyush-garg thanks for the patch, could you please add a test?

File: pkg/cmd/root.go [hunk 1]
```diff
@@ -15,6 +15,7 @@
 package cmd
 
 import (
+	"fmt"
```
Comment (vdemeester) [inline pkg/cmd/root.go]: nit: imports should be grouped :angel: 

```go
import (
	"fmt"
	"strings"

	"github.com/spf13/cobra"
	# …
)
```

**Comment by piyush-garg** [comment]
@sthaha @vdemeester Test added.

File: pkg/cmd/root_test.go [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+)
+
+func TestCommand_invalid(t *testing.T) {
+	p := &test.Params{}
+	pipelinerun := Root(p)
+	out, err := test.ExecuteCommand(pipelinerun)
+	if err == nil {
+		t.Errorf("No errors was defined. Output: %s", out)
+	}
+	expected := "Use one of available subcommands: completion [SHELL], help [command], pipeline, pipelinerun, task, taskrun, version"
```
Comment (sthaha) [inline pkg/cmd/root_test.go]: nit: we may not want to check the exact output as adding a new command means changing this test.

**Pull request merged**